content/applications/sales/point_of_sale/payment_methods/terminals/mercado_pago.rst

```diff
@@ -0,0 +1,101 @@
+============
+Mercado Pago
+============
+
+Connecting a payment terminal allows you to offer a fluid payment flow to your customers and ease
+the work of your cashiers.
+
+.. important::
+   - Odoo only supports **Point Smart** payment terminals. They can be purchased on
+     `Mercado Pago's website <https://www.mercadopago.com.mx/herramientas-para-vender/lectores-point>`_.
+   - Point Smart can only be used in the following :abbr:`LATAM (Latin America)` countries:
+
+     - Argentina
```

Comment:
@LoredanaLrpz the scope is actully just Mexico, Argentina and Brazil, can you update it please?